app/src/main/java/io/github/droidkaigi/confsched/fragment/SessionsTabFragment.java

```diff
@@ -152,6 +154,7 @@ public void onBindViewHolder(BindingHolder<ItemSessionBinding> holder, int posit
                 public void liked(LikeButton likeButton) {
                     session.checked = true;
                     dao.updateChecked(session);
+                    AlarmUtil.registerSessionAlarm(getActivity(), session);
```

Comment:
Like button also exists in SessionDetailFragment.
You have to write this logic to detail, right?